public/components/common/custom-search-bar/components/multi-select.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Wazuh app - React component Multi-select
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  EuiFieldSearch,
+  EuiFilterButton,
+  EuiFilterGroup,
+  EuiFilterSelectItem,
+  EuiIcon,
+  EuiLoadingChart,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiSpacer,
+  FilterChecked,
+} from '@elastic/eui';
+import { IValueSuggestion, useValueSuggestion } from '../../hooks';
+
+const ON = 'on';
+const OFF = 'off';
+
+export const MultiSelect = ({ item, onChange, selectedOptions, onRemove }) => {
```

Comment:
This component should have a similar behaviour as the one implemented to the Combobox in [this PR](https://github.com/wazuh/wazuh-kibana-app/pull/3529).
Changing the component used in the search bar from the Combobox to MultiSelect may have been outside the scope of the issue, but if we are keeping this decision we should make sure it keeps an expected behavior, and that means that selecting several options should return values that match any of the selected rather than all of them at once, because any 2 options in a single field are mutually exclusive.